src/app/core/services/github.service.ts

```diff
@@ -58,12 +59,24 @@ export class GithubService {
   private issuesLastModifiedManager = new IssueLastModifiedManagerModel();
   private issueQueryRefs = new Map<Number, QueryRef<FetchIssueQuery>>();
 
-  constructor(private errorHandlingService: ErrorHandlingService, private apollo: Apollo, private electronService: ElectronService) {}
+  constructor(
+    private errorHandlingService: ErrorHandlingService,
+    private apollo: Apollo,
+    private electronService: ElectronService,
+    private logger: LoggingService
+  ) {}
 
   storeOAuthAccessToken(accessToken: string) {
     octokit = new Octokit({
       auth() {
         return `Token ${accessToken}`;
+      },
+      log: {
+        debug: (message, ...otherInfo) => this.logger.debug(message, ...otherInfo),
+        // Do not log info for HTTP response 304 due to repeated polling
+        info: (message, ...otherInfo) => (/304 in \d+ms$/.test(message) ? undefined : this.logger.info(message, ...otherInfo)),
```

Comment:
I think this should be fine for now ba, but need to be careful cause regex checking takes q a while. Any comments mr @dingyuchen ?

Reply:
I believe this is fine for now as well. If performance is an issue, we can simply use `String.includes("304 in")` rather than regex. I think it should not be an issue given the polling frequency.